analytic_engine/src/sampler.rs

```diff
@@ -292,7 +292,7 @@ impl PrimaryKeySampler {
             .iter()
             .enumerate()
             .map(|(idx, col)| {
-                if idx == timestamp_index {
+                if col.data_type.is_timestamp() {
```

Comment:
It is possible that, `Timestamp` typed column but not a `time key` is suggested to be one of the order keys?

Reply:
It make no sense for me, I think we should be strict at first.